src/js/step_definitions/1-LicenceEntry/I_submit_the_licence_###_and_postcode_###.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+const { defineStep } = require('@cucumber/cucumber')
+const LoginPage = require('../../pages/Licence-page')
+const { logger } = require('defra-logging-facade')
+
+defineStep(/I submit the licence (.*) and postcode (.*)/, async (licence, postcode) => {
+  logger.debug(`Attempting to login with licence:${licence} and postcode:${postcode}`)
```

Comment:
```suggestion
  logger.debug(`Attempting to login with licence: ${licence} and postcode: ${postcode}`)
```

Think this would be a bit easier to read